frontend/src/components/worksheets/BundleDetail/SideBar.jsx

```diff
@@ -65,82 +62,181 @@ class SideBar extends React.Component<
+                                    href={ `/worksheets/${ worksheet.uuid }`}
+                                    className={ classes.uuidLink }
+                                    target="_blank"
+                                >
+                                    { worksheet.name }
+                                </a>
+                            </div>
+                        )
+                    :   <div>None</div>
                 }
                 {/** ----------------------------------------------------------------------------------------------- */}
-                <div className={classes.spacer}/>
                 <div>
-                    <Typography variant="subtitle1">Permissions</Typography>
-                    <PermissionDialog
-                        uuid={ bundleInfo.uuid }
-                        permission_spec={ bundleInfo.permission_spec }
-                        group_permissions={ bundleInfo.group_permissions }
-                        onChange={ this.props.onMetaDataChange }
-                    />
+                    <ConfigLabel label="Permissions:" inline={true}/>
+                    <div
+                        onClick={() => { this.setState({ showPermisson: !this.state.showPermisson}) }}
+                        className={classes.permissions}
+                    >   
+                        <Grid container>
+                            <Grid inline style={{ paddingTop: 2 }}>
+                                {renderPermissions(bundleInfo)}
+                            </Grid>
+                            <Grid inline>
+                                <Icon fontSize={'small'}>
+                                    <KeyboardArrowDownIcon />
+                                </Icon>
+                            </Grid>
+                        </Grid>
+                    </div>
+                    {this.state.showPermisson
+                        ?   <div>
+                                <PermissionDialog
+                                    uuid={bundleInfo.uuid}
+                                    permission_spec={bundleInfo.permission_spec}
+                                    group_permissions={bundleInfo.group_permissions}
+                                    onChange={this.props.onMetaDataChange}
+                                    wperm
+                                />
+                            </div>
+                        :   null
+                    }
                 </div>
+                {/** ----------------------------------------------------------------------------------------------- */}
                 <div>
                     <a
                         href={ `/bundles/${ bundleInfo.uuid }` }
                         className={ classes.uuidLink }
                         target="_blank"
                     >
-                        More Information for Bundle { shorten_uuid(bundleInfo.uuid) }
+                        View Bundle details

```

Comment:
```suggestion
                        View bundle details
```

Does this really have more information that isn't already present in the new bundledetail view?

Reply:
License, tags, source url?